charts/pega/README.md

```diff
@@ -672,6 +672,15 @@ Parameter     | Description | Default value
 `trustStorePassword` | If required provide trustStorePassword value in plain text. For better security leave this parameter blank and include the password in an external secrets manager with the key CASSANDRA_TRUSTSTORE_PASSWORD. | Empty
 `keyStore` | If required, provide the keystore certificate file name.<br/>When using a keystore certificate, you must also include a Kubernetes secret name that contains the keystore certificate in the global.certificatesSecrets parameter.<br/>Pega deployments only support keystores that use the Java Key Store (.jks) format. | Empty
 `keyStorePassword` | If required provide keyStorePassword value in plain text. For better security leave this parameter blank and include the password in an external secrets manager with the key CASSANDRA_KEYSTORE_PASSWORD. | Empty
+`asyncProcessingEnabled*` | Specify a prefix to use when creating Pega-managed keyspaces in Cassandra. | false
+`keyspacesPrefix` | Specify a prefix to use when creating Pega-managed keyspaces in Cassandra. |
+`extendedTokenAwarePolicy*` | Enable an extended token aware policy for use when a Cassandra range query runs. When enabled this policy selects a token from the token range to determine which Cassandra node to send the request. Before you can enable this policy, you must configure the token range partitioner. | false
+`latencyAwarePolicy*` | Enable a latency awareness policy, which collects the latencies of the queries for each Cassandra node and maintains a per-node latency score (an average). | false
+`customRetryPolicy*` | Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy in your deployment configuration, Cassandra queries that timeout will be retried. The number of retries may be configured using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. If not configured, queries will be retried once. | false
```

Comment:
avoid passive voice by using active voice>>
Enable the use of a customized retry policy for your Pega Platform deployment. After you enable this policy in your deployment configuration, the deployment retries Cassandra queries that timeout. Configure the number of retries using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. The default is 1, so if you do not specify a retry count, timed out queries are retried once.

Reply:
Fixed